common/src/test/resources/clinical/patient.clinical.json

```diff
@@ -120,7 +120,7 @@
   "complications": [
     {
       "name": "Nausea",
-      "categories": "Other",
+      "categories": ["Other"],
```

Comment:
The formatting is inconsistent, it should be something like 
[
   "Other"
]

I generally use auto-formatting in IntelliJ for json files, works acceptably well in my opinion.